### ISSUE TITLE
Build Update Settings Form for Governance and Tokens

### ISSUE THREAD

**Issue by earth2travis** [HausDAO/daohaus-monorepo#623]
This ticket covers two proposal forms, for governance settings and for token transferability. 

## Both/Shares Fields
- Basics (should be the same as all other proposal forms): 
    - Proposal Name (required) 
    -  Description
    - Link
 - All others > listed below
- Expiration (should be same as all other proposals, see https://github.com/HausDAO/daohaus-monorepo/issues/498 
- Submit button > proposal created 

## Governance Update Form Fields

Governance Update Form
https://www.figma.com/file/9lCwprCNaoAmUDmiMQpFEA/DAOhaus-v3?node-id=6852%3A192157

- Proposal Timing (h5)
- Update your timing for Voting and Grace periods. (text)
- Voting Period 
    - Info dot + input = same as summon
- Grace Period 
    - Info dot + input = same as summon
- Advanced Governance (h5)
- Modify some advanced governance features. (text)
    - Quorum % (same as summon, placeholder = current setting)
    - Min Retention % (same as summon, placeholder = current setting)
    - Sponsor Threshold  (same as summon, placeholder = current setting)
    - New Offering (same as summon, placeholder = current setting)

## Token Transferability Form Fields

Token Transferability Form
https://www.figma.com/file/9lCwprCNaoAmUDmiMQpFEA/DAOhaus-v3?node-id=5694%3A125071

- DAO Tokens (h5)
- Update your token transferability. (text)
- Voting Stake Transferability? (switch - same as summon)
    - Info dot: (same as summon) 
- Non-Voting Stake Transferability? (switch - same as summon)
    - Info dot: (same as summon)   

Notes:
- This will use the transaction pending pattern: https://github.com/HausDAO/daohaus-monorepo/issues/17 
- Once submitted, land user back on the settings screen 
- Proposal details will have a 'governance update' specific message on it. Another ticket will be filed for this.  

### Transaction details

This will call the baal submitProposal
```
    function submitProposal(
        bytes calldata proposalData,
        uint32 expiration,
        uint256 baalGas,
        string calldata details
    ) 
```
- expiration: timestamp
-baalGas: gwei
- details - is the json detail string
  - include type: 'issueFunding' or something

#### Proposal action data for governance settings:

contract: baal
function: setGovernanceConfig
args:

uint32 voting
uint32 grace
uint256 newOffering
uint256 quorum
uint256 sponsor
uint256 minRetention

#### Proposal action data for token transfers:
`setAdminConfig(bool pauseShares, bool pauseLoot)`

contract: baal
function: setAdminConfig
args:

bool pauseShares
bool pauseLoot

**Comment by earth2travis** [comment]
@adriennetilley

**Comment by adriennetilley** [comment]
Designs ready for review.
